src/Microsoft.Health.Fhir.Shared.Core/Features/Search/Expressions/Parsers/SearchValueExpressionBuilderHelper.cs

```diff
@@ -338,8 +338,14 @@ private Expression GenerateNumberExpression(FieldName fieldName, decimal number)
 
             switch (_comparator)
             {
-                case SearchComparator.Eq:
                 case SearchComparator.Ap:
+                    var approximateModifier = Math.Abs(number * Convert.ToDecimal(ApproximateMultiplier));
+                    lowerBound -= approximateModifier;
+                    upperBound += approximateModifier;
+                    return Expression.And(
```

Comment:
Could probably use a `goto case SearchComparator.Eq;` here after modifying the lower and upper bound.